cephfs/path_test.go

```diff
@@ -399,3 +399,52 @@ func TestRename(t *testing.T) {
 		assert.NoError(t, mount.RemoveDir(n2))
 	})
 }
+
+func TestTruncate(t *testing.T) {
```

Comment:
Just curious, what will happen if we pass the path to a directory? Should we add a test case.

Reply:
Ceph should error (with EBADF IIRC). I don't think it makes sense to cover every condition ceph libs _should_ check for. That said, if we find an issue we should file it like I did for the truncate-read-only-fd issue, which I stumbled on by accident.